tests/core/array/test_basic.py

```diff
@@ -15,7 +15,7 @@
 
 
 # pylint: disable=wrong-import-order
-import common
+import common  # pylint: disable=import-error
```

Comment:
Does this directive only apply to this line?

Reply:
@bveeramani bumping this.

Reply:
Yes, it should only apply to that line

Reply:
I'm like 95% sure but I can double-check.